htdocs/comm/propal/list.php

```diff
@@ -635,16 +635,29 @@
 if ($search_availability) {
 	$sql .= " AND p.fk_availability IN (".$db->sanitize($db->escape($search_availability)).')';
 }
+$societe_add_ref_in_list = (!empty(getDolGlobalString('SOCIETE_ADD_REF_IN_LIST')) ? getDolGlobalString('SOCIETE_ADD_REF_IN_LIST') : 0); // Récupérer la valeur de SOCIETE_ADD_REF_IN_LIST
```

Comment:
`$societe_add_ref_in_list = getDolGlobalInt('SOCIETE_ADD_REF_IN_LIST');` is more simple and pass the tests

Reply:
It's ok now